utils/build/virtual_machine/weblogs/nodejs/provision_test-app-nodejs.yml

```
@@ -1,25 +1,25 @@
 lang_variant:
-    name: node16
-    version: 16
+    name: node18
+    version: 18
     cache: true
     install: 
       - os_type: linux
         os_distro: deb
-        remote-command: curl -sL https://deb.nodesource.com/setup_16.x | sudo bash - && sudo apt-get install -y nodejs  
+        remote-command: curl -sL https://deb.nodesource.com/setup_18.x | sudo bash - && sudo apt-get install -y nodejs  
       - os_type: linux
         os_distro: rpm
         remote-command: |
           curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
```

How about we update this too to the latest while we're here?
```suggestion
          curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
```

---

Done!